metadata-ingestion/src/datahub/ingestion/source/looker/lookml_sql_parser.py

```diff
@@ -109,7 +125,11 @@ def sql_query(self):
             sql_query = f"{sql_query} FROM {self.view_name}"
             # Get the list of tables in the query
 
-        return sql_query
+        # set liquid variables value to NULL
+        Undefined.__str__ = lambda instance: "NULL"  # type: ignore
+
+        # Resolve liquid template
+        return Template(sql_query).render(self.liquid_context)
```

Comment:
the user should also be able to pass in a variable list into the ingestion config

Reply:
Updated the code to pass liquid_variable as config